templates/webapi/test/infopanel.html.ep

```diff
@@ -64,20 +64,44 @@
                         <abbr class="timeago" title="<%= $job->t_started->datetime() %>Z"><%= format_time($job->t_started) %></abbr>
                     % }
                     % if (is_operator && $job->can_be_duplicated) {
+                      <div class = "btn-group btn-group-sm">
+                        <button class="btn btn-link" type="button" id="restart-button-default">
                         %= link_post url_for('apiv1_restart', jobid => $testid) => ('data-remote' => 'true', id => 'restart-result', 'data-jobid' => $testid) => begin
                             <i class="fa fa-2 fa-undo" title="Restart job"></i>
                         %= end
-                        <%= help_popover('Restart job' => '
-                            <p>Restarts the job and certain dependent jobs.</p>
-                            <p>Rules for restarting dependencies:</p>
-                            <ul>
-                                <li>If the job is part of a parallel cluster, all jobs within that cluster are restarted.</li>
-                                <li>Any kind of chained child jobs are restarted.</li>
-                                <li>Directly chained parent jobs are restarted.</li>
-                            </ul>
-                            All rules are applied recursively. Direct use of the REST-API allows excluding parents and (passed/softfailed) children.',
-                            'https://open.qa/docs/#_handling_of_related_jobs_on_failure_cancellation_restart', 'documentation', 'bottom');
-                        %>
+                        % if ($job->has_dependencies) {
+                        <button class="btn btn-link dropdown-toggle" type="button" id="restart-button-options" data-toggle="dropdown">
+                        <span class="caret" title="Additional options for restarting jobs with dependencies"></span></button>
+                          <ul class="dropdown-menu" role="menu" aria-labelledby="restart-button-options" id="restart-menu">
+                            <li role="presentation">
+                              %= link_post url_for('apiv1_restart', jobid => $testid)->query({skip_parents => 1}) => ('data-remote' => 'true', id => 'restart-result-skip-parents', 'data-jobid' => $testid, role => 'menuitem') => begin
```

Comment:
The links might be missing `class="dropdown-item"` for the right styling. 🤔

Reply:
Fixed in https://github.com/os-autoinst/openQA/pull/4425